JetLagged/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*.{kt,kts}]
```

Comment:
WDYT about commenting why we're adding these exceptions? I'm assuming it's so that the different linters / checkers are in sync?